src/lib/containers/download_list_v2/DownloadListStats.tsx

```diff
@@ -1,43 +1,24 @@
-import React, { useEffect } from 'react'
-import { useErrorHandler } from 'react-error-boundary'
-import { toError } from '../../utils/ErrorUtils'
-import { SynapseSpinner } from '../LoadingScreen'
-import { useGetDownloadListStatistics } from '../../utils/hooks/SynapseAPI/useGetDownloadListStatistics'
+import React from 'react'
 import DownloadDetails from './DownloadDetails'
 
-export default function DownloadListStats() {
-  const handleError = useErrorHandler()
-  const {
-    data,
-    isFetching,
-    isError,
-    error: newError,
-  } = useGetDownloadListStatistics()
-  useEffect(() => {
-    if (isError && newError) {
-      handleError(toError(newError))
-    }
-  }, [isError, newError, handleError])
+export type DownloadListStatsProps = {
```

Comment:
Now the stats component just takes the number of files and the byte value as props.